speechbrain/nnet/quaternion_networks/q_CNN.py

```diff
@@ -363,6 +390,12 @@ class QConv2d(torch.nn.Module):
         with deep configurations. The vector_scale parameters are learnable
         parameters that acts like gates by multiplying the output vector with
         a small trainable parameter (default False).
+    max_norm: float
+        kernel max-norm.
+    swap: bool
+        If True, the convolution is done with the format (B, C, W, H).
+        If False, the convolution is dine with (B, H, W, C).
```

Comment:
dine -> done

Reply:
How does this relate to skip_transpose?

Reply:
Good point, I could add the skip_transpose option for better consistency with the regular Conv2d layers.